### DISCUSSION TITLE
Bad search indexes?

### DISCUSSION THREAD

**Discussion by mpachas** [djangopackages/djangopackages#658] [General]
I don't know why when I search for "static" term I don't get whitenoise as an answer.. Maybe it's just an indexing bug.

**Comment by ryancheley** [comment]
The search tool looks at package title or slug, but not the description (see https://github.com/djangopackages/djangopackages/blob/a2aeb53a5d7e2facf4de3fba019baa9901ab8915/searchv2/views.py#L31)
